backend/tools/get_slots_in_week.py

```diff
@@ -0,0 +1,14 @@
+from db.models import Slots, SlotsShema
+from datetime import timedelta
+from tools.transform_str_to_datetime_and_back import transform_string_to_datetime, transform_datetime_to_string
+
+
+def get_slots_in_week(date):
+    """Return list of slots for the week starting from <date>
+    """
+    end_interval = transform_datetime_to_string(transform_string_to_datetime(date) + timedelta(days=7))
+
+    list_of_slots_in_week = Slots.query.filter(Slots.start_interval.between(date, end_interval)).all()
```

Comment:
need to add filters: or_ Slots.end_interval.beetween(date, end_interval) or_ (Slots.start_interval <= date and_ Slots.end_interval >= end_interval)